TYPO3.Fluid/Classes/TYPO3/Fluid/Core/Parser/SyntaxTree/ObjectAccessorNode.php

```diff
@@ -88,10 +88,22 @@ public static function getPropertyPath($subject, $propertyPath, RenderingContext
     {
         $propertyPathSegments = explode('.', $propertyPath);
         foreach ($propertyPathSegments as $pathSegment) {
+            $isHasDirectAccess = false;
             try {
-                $subject = ObjectAccess::getProperty($subject, $pathSegment);
+                if (preg_match('/^(is|has)([A-Z].*)/', $pathSegment, $matches) > 0) {
```

Comment:
Nevermind - as you're only comparing the first letter after "is"/"has" this is not a real problem!